views/login.handlebars

```diff
@@ -107,21 +174,29 @@
       <p class="details">You are currently logged in to Clay</p>
     {{else}}
       <h1 class="header">Log in to Clay</h1>
-      <p class="details">Please choose the provider attached to your account</p>
+      {{#if currentProviders.length}}<p class="details">Please choose the provider attached to your account</p>{{/if}}
     {{/with}}
-    <div class="buttons">
+    <div class="login-options">
       {{#if user}}
         <a class="button logout" href="{{ logoutLink }}" title="Log Out">
           <div class="button-icon">{{> logout }}</div>
           <span class="button-text">Log out</span>
         </a>
       {{else}}
-        {{#each currentProviders }}
-          <a class="button {{this.name}}" href="{{ this.url }}" title="{{ this.title }}">
-            <div class="button-icon">{{> ( icon ) }}</div>
-            <span class="button-text">{{ this.title }}</span>
-          </a>
+        {{#each currentProviders}}
+        <div class="buttons">
```

Comment:
This `div` is wrapper by the `each` so, please add an indentation level to this element.